actions/bump-version/action.yaml

```diff
@@ -18,15 +18,23 @@ runs:
     - name: 'Get next version'
       id: get_version
       shell: bash
-      run: echo "version=$(cz bump --get-next)" >> "$GITHUB_OUTPUT"
+      run: |
```

Comment:
The previous version of this step was not taking into account the `prerelease` input, so it would output `0.12.0` even when the version was being bumped to the beta counterpart: `0.12.0b0`.